api-reference/beta/api/alert_get.md

```diff
@@ -0,0 +1,205 @@
+# Get alert
+
+Retrieve the properties and relationships of an [alert](../resources/alert.md) object.

```

Comment:
Let's make the first mention of the resource name a link to the resource topic. Subsequent mentions of the resource name should be bold.